src/cmd/linuxkit/run_vcenter.go

```diff
@@ -191,6 +143,66 @@ func runVcenter(args []string) {
 	}
 }
 
+func vCenterConnect(ctx context.Context, newVM vmConfig) (*govmomi.Client, *object.Datastore, *object.DatacenterFolders, *object.HostSystem, object.NetworkReference, *object.ResourcePool) {
+
+	// Parse URL from string
+	u, err := url.Parse(*newVM.vCenterURL)
+	if err != nil {
+		log.Fatalf("URL can't be parsed, ensure it is https://username:password/<address>/sdk")
+	}
+
+	// Connect and log in to ESX or vCenter
+	c, err := govmomi.NewClient(ctx, u, true)
+	if err != nil {
+		log.Fatalln("Error logging into vCenter, check address and credentials")
```

Comment:
same comment re: including error message: `log.Fatalf("Error logging into vCenter, check address and credentials: %v", err)`